hack/common.sh

```diff
@@ -61,3 +61,14 @@ function kubevirt_version() {
     fi
 }
 KUBEVIRT_VERSION="$(kubevirt_version)"
+
+determine_cri_bin() {
+    if curl --unix-socket "${XDG_RUNTIME_DIR}/podman/podman.sock" http://d/v3.0.0/libpod/info >/dev/null 2>&1; then
+        echo podman
+    elif docker ps >/dev/null 2>&1; then
+        echo docker
+    else
+        echo "error: no docker or podman found"
+        exit 1
+    fi
+}
```

Comment:
Aren't we doing this at kubevirtci ? 

https://github.com/kubevirt/kubevirtci/blob/23ddfc29c03bb1ccaeb79cd204163f8108db1c36/cluster-up/cluster/ephemeral-provider-common.sh#L17-L36

Reply:
yes but there it affect only cluster-up scope, not cluster-sync and the calls of the e2e / unit tests and so on